x-pack/plugins/security_solution/public/management/pages/policy/view/trusted_apps/layout/policy_trusted_apps_layout.tsx

```diff
@@ -164,8 +163,10 @@ export const PolicyTrustedAppsLayout = React.memo(() => {
               policyName={policyItem.name}
             />
           )
+        ) : displayHeaderAndContent ? (
+          <PolicyTrustedAppsList />
         ) : (
-          <PolicyTrustedAppsList hideTotalShowingLabel={true} />
+          <EuiProgress size="xs" color="primary" />
```

Comment:
This looks strange here to show a progress bar all by its self.

if we try to mirror what is done in Trusted Apps list page or with Eui's Tables, we should show a progress bar right above the body of the grid (below the grid's header). And the `ArtifactCardGrid` should probably support a `loading` prop that would show/hide it.

Reply:
This is for when we are checking if there are assigned TA to the policy (to ensure if we have to display the empty state or the grid itself).

Reply:
ahhh. ok. but should it be a Loading spinner instead of the progress bar?

Reply:
There is a progress bar in event filters and host isolation list, wondering what should be the correct one